### PR TITLE
browserify and webpack should be bundlers

### LINKED ISSUE #139: browserify should be npm as the title is "Package Managers"
browserify - Browser-side require() the node.js way.
`browserify` should probably be `npm`. `npm` is the package manager and `browserify` is the build tool.
I would reword this to
npm - when built with browserify, you can require() the node.js way in browser.

### PR DESCRIPTION
**Pull request by stevemao**
Add npm to package manager

Fixes #139

### CODE REVIEW

**Comment by huei90** [comment]
Looks great :beers:

**Comment by stevemao** [comment]
Thanks @huei90

**Comment by huei90** [comment]
Thanks for your contribution :smile:
